docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,131 @@
-
-- To set up each tool path directly:
-  1. In the Visual Studio Mac IDE menu, go to **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-  2. Enter the root directory of your existing Tizen Studio installation in the **Tool Path** field.
-
-     ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-     The other tools paths are automatically set up.
-
-### Install Emulator Images
-
-If you do not have a real device, you can run applications in the Tizen Emulator.
-
-To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager:
-
-- To use the Tizen Package Manager:
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-  2. Select the profiles and versions you want to install and click **Install**.
-
-     ![Package Manager](media/howtoinstall-packagemanager.png)
-
-- To use the Tizen Emulator Manager:
-
-  > **Note**
-  >
-  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
-
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
-  2. Select the profiles and versions you want to install and click **OK**.
-
-     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
-
-
-## Troubleshoot
-
-If you encounter problems with the installation:
-
-- To make sure the baseline Tizen SDK is installed correctly, check the tool path by selecting **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-
-  ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-
+# Install Visual Studio for Mac Extension for Tizen.
+
+Visual Studio for Mac Extension for Tizen is an extension, which enables you to develop Tizen .NET applications easily using various Tizen project templates with Visual Studio for Mac.
+
+## Prerequisites
+
+To work with Visual Studio for Mac Extension for Tizen, your computer must have:
+
+- At least 5.6 GB of available disk space
+- macOS Mojave version 10.14 (Community, Professional, and Enterprise)
+- Visual Studio 2019 for Mac version 8.2 and higher (Community, Professional, and Enterprise)
+- Java Development Kit (JDK)
+
+  > [!NOTE]
+  > To use Tizen Baseline SDK, you must install [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Java for OS X 2015-001](https://support.apple.com/kb/DL1572) together or [OpenJDK 12 and OpenJFX Installation Guide](../tizen-studio/setup/openjdk.md#install-openjdk-for-macos).
+  > 
+  > Ensure that you download and install the exact version. Additionally, for Oracle JDK, Java for OS X 2015-001 is required to support the legacy Java software on macOS.
+
+## Install Visual Studio for Mac Extension for Tizen
+
+To use the Tizen tools in Visual Studio for Mac, you must install:
+
+1. [Visual Studio for Mac Extension for Tizen](#install-visual-studio-for-mac-extension-for-tizen )

```

Comment:
Just for the reference. As the heading on line 26 is same with the title of this page,
when I click this link, it is jumping to the starting point of this page.